pyproject.toml

```diff
@@ -8,17 +8,17 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = "^3.6"
 py-buzz = "^2.0.0"
-flask = "^1.0"
+flask = ">=1.0, !=2.0.0"
 
 [tool.poetry.dev-dependencies]
-flask-restx = "^0.4.0"
+flask-restx = ">=0.4"
 flake8 = "^3.7"
 pytest = "^6.0"
 pytest-flask = "^1.1.0"
 toml = "^0.10.0"
 black = "^20.8b1"
 isort = "^5.7.0"
-werkzeug = "0.16.1"
+werkzeug = "!=2.0.0"
```

Comment:
Not too sure about this change. Was there a reason for werkzeug being fixed to 0.16.1 previously?

Reply:
Yeah, it was breaking on later versions. Thanks for your fix, I'll merge it

Reply:
Actually, we should just remove this dependency altogether

Reply:
Okay, I have removed it now.